tasks/Linux/selinux-support.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: "Install ansible selinux support library"
+  package:
+    name: libselinux-python
+    state: present
+  become: True
+
+- name: "Install ansible selinux configure libraries"
+  package:
+    name: "{{ selibs_item }}"
+    state: present
+  with_items:
```

Comment:
It's deprecated type of package install. Please use passing a list to name through variable or directly:
```yaml
package:
    name:
      - policycoreutils-python
      - libsemanage-python
```